bin/git-cml

```diff
@@ -173,15 +65,30 @@ def add(args):
     """
     Splits model parameters file into parameter groups on filesystem and stages files
     """
-    repo = get_git_repo()
-    model_path = create_git_cml_model_dir(repo, args.file)
+    repo = git_utils.get_git_repo()
+    model_path = git_utils.create_git_cml_model_dir(repo, args.file)
 
-    param_dict = load_file(args.file)
+    param_dict = git_utils.load_tracked_file(args.file)
     for (param, param_file) in iterate_dict_and_subdirs(param_dict, model_path):
-        write_file(param_file, param)
-        add_file(param_file, repo)
+        git_utils.write_tracked_file(param_file, param)
+        git_utils.add_file(param_file, repo)
+
+    git_utils.add_file(args.file, repo)
 
-    add_file(args.file, repo)
+
+def init(args):
+    """
+    Set up git-cml repo by initializing .git/info/attributes and .git/config
+    """
+    repo = git_utils.get_git_repo()
+    config_writer = repo.config_writer()
+    config_writer.set_value('filter "cml"', 'clean', 'git-cml-filter clean %f') 
+    config_writer.set_value('filter "cml"', 'smudge', 'git-cml-filter smudge %f')
+    config_writer.set_value('filter "cml"', 'required', 'true')
+    config_writer.release()
+    
+    with open(os.path.join(repo.working_dir, '.git/info/attributes'), 'w+') as f:
+        f.write('*.json filter=cml')
```

Comment:
Something to discuss later - we need to handle the fact that PyTorch checkpoints have no standard extension (sometimes it's .pt, sometimes it's .bin, sometimes it's something else, but there's no standardization)